js/index.js

```diff
@@ -1,42 +1,146 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
 
   //... your code goes here
-}
+  //capture the price and quantity values, and convert to numbers
+  const price = product.querySelector('.price span').innerHTML;
+  const quantity = product.querySelector('.quantity input').value
+
+  //Get the subtotal value
+  const subTotal = Number(price) * parseFloat(quantity)
+
+//REnder the subtotal value in the element with class ".subtotal"
+  const subTotalElement = product.querySelector(".subtotal span")
+  //modify innr text
+  subTotalElement.innerHTML = subTotal;
+  return subTotal;
+} 
+
 
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
+  // const singleProduct = document.querySelectorAll('.cartItem');
+  // const products = document.getElementsByClassName('.cartItem')
+  // updateSubtotal(singleProduct);
   // end of test
 
   // ITERATION 2
   //... your code goes here
+  
+  //call the functioin updateSubtotal with every tr.product DOM nodde in the table caart 
+  // get cart item elements by .getElementByClassName
```

Comment:
After you're done, I'd recommend removing any code/comments that no longer serves any use to keep your code minimal and easier to read.